ros/models.py

```diff
@@ -6,12 +6,16 @@
 
 
 class PerformanceProfile(db.Model):
-    __table_args__ = (db.UniqueConstraint('inventory_id', 'report_date'), )
     id = db.Column(db.Integer, primary_key=True)
     inventory_id = db.Column(UUID(as_uuid=True), nullable=False)
```

Comment:
We can remove this column, since we have it in system table. Otherwise looks good :+1: .

Reply:
Makes sense, but, removing this would affect the API call `/systems` [here](https://github.com/RedHatInsights/ros-backend/blob/main/ros/resources/v0/hosts.py#L73-L86). So, this should not be merged until the API call is fixed.
Thoughts?

Reply:
Ok, then I think this can be merged

Reply:
You mean merged with `inventory_id` present in `performance_profile` table, or without?

Reply:
Removed, the app will still be nonfunctional until we implement changes in the processor and the API.